README.md

```diff
@@ -69,12 +69,13 @@ and it is always recommended keeping Drupal core installations up to date.
 
 ## Dependencies
 
-This module relies on the [GNU gettext toolset](https://www.gnu.org/software/gettext/) & the [Symfony Process Component](https://symfony.com/doc/current/components/process.html).
+This module relies on the [GNU gettext toolset](https://www.gnu.org/software/gettext/), the [Symfony Process Component](https://symfony.com/doc/current/components/process.html) & the [Symfony Finder Component](https://symfony.com/doc/current/components/finder.html).
 
 * `Symfony Process Component` is an external PHP library to execute binaries ;
+* `Symfony Finder Component` is an external PHP library to execute binaries ;
```

Comment:
* `Symfony Process Component` is an external PHP library to execute commands in sub-processes.
* `Symfony Finder Component` is an external PHP library to finds files and directories via an intuitive fluent interface.